thorlcr/master/thmastermain.cpp

```diff
@@ -904,38 +911,38 @@ int main( int argc, const char *argv[]  )
         if (isEmptyString(graphName))
             throw makeStringException(0, "missing --graphName");
 
-        if (!globals->hasProp("@numSlaves"))
-            throw makeStringException(0, "Default number of slaves not defined (numSlaves)");
+        if (!globals->hasProp("@numWorkers"))
+            throw makeStringException(0, "Default number of workers not defined (numWorkers)");
         else
         {
-            // check 'numSlaves' workunit option.
+            // check 'numWorkers' workunit option.
```

Comment:
Is this a cloud-only feature (so we don't need to worry about back-compatibility?)

Reply:
yes, container only. No such equivalent in bare-metal. The number of slaves is ascertained differently (not a property value)

Reply:
The number of workers, you mean...

Reply:
that's what I said :) (edit)